mod_reforged/hooks/config/tip_of_the_day.nut

```diff
@@ -15,6 +15,9 @@ for (local index = (::Const.TipOfTheDay.len() - 1); index >= 0; index--)
         case "Losing is fun.":
         case "Some people will use you and throw you away.":
             ::Const.TipOfTheDay.remove(index);
+			break;
+		case "Flails ignore the defense bonus of shields.":
+			::Const.TipOfTheDay[index] = "Flails and Bolas ignore only the defense bonus of shields. The defense granted by \'Shieldwall\' still applies.";
```

Comment:
Would be good to make it shorter so it's faster to read on a loading screen. Something like `Flails and Bolas ignore the defense granted by shields, but not by \'Shieldwall\'`.